plugins/pullvals.py

```diff
@@ -42,7 +42,11 @@ def pullvals(histpair,
     # Normalize data_hist
     if norm_type == "row":
         normalize_rows(data_hist, ref_hist)
-    else:
+    elif norm_type == "column":
+        normalize_cols(data_hist, ref_hist)
+    elif norm_type == "total":
+        normalize_total(data_hist, ref_hist)
```

Comment:
This should be covered by the else statement below. In fact the normalize_total function that you created scales the reference histogram which we would like to avoid considering the reference runs should, in general, have more stable statistics. 

Please remove this elif and the associated normalize_total function. The rest of the PR looks healthy

Reply:
Sure, just pushed a commit removing the total norm option.